packages/core/src/modules/credentials/formats/jsonld/JsonLdCredentialFormatService.ts

```diff
@@ -86,38 +84,30 @@ export class JsonLdCredentialFormatService extends CredentialFormatService<JsonL
    * @param options the options needed to accept the proposal
    */
   public async processProposal(agentContext: AgentContext, { attachment }: FormatProcessOptions): Promise<void> {
-    const credProposalJson = attachment.getDataAsJson<SignCredentialOptionsRFC0593>()
+    const credProposalJson = attachment.getDataAsJson<JsonLdSignCredentialFormat>()
```

Comment:
We should be careful here. The `JsonLdSignCredentialFormat` is for the input (the API) we use in AFJ. Although this is currently the same as the RFC, this doesn't necesarily stay the same. I think we shouldn't reuse interfaces for different purposes like this, as it increases the chance of having invalid interfaces in some places of the codebase.

Reply:
I have put back the SignCredentialOptionsRFC0593 for use internally (only) so if there is a divergence we are covered.

I assume that is what we want to do here, please let me know if there is a better way of fixing this.